manifests/install.pp

```diff
@@ -46,10 +44,6 @@
         ensure => $nomad::package_ensure,
       }
 
-      if $nomad::manage_user {
-        User[$nomad::user] -> Package[$nomad::package_name]
```

Comment:
This feels odd. If the package creates the user, wouldn't you prefer it to be the other way around? I know it's actually removing it, but still wanted to note it.

Reply:
to me this was a legacy from when hashicorp didnt make native packages, as noted above upstream packages runs as root so no user management is required